pkgs/cupertinohttp/third_party/native_library/.gitignore

```diff
@@ -0,0 +1,48 @@
+# Miscellaneous
+*.class
+*.log
+*.pyc
+*.swp
+.DS_Store
+.atom/
+.buildlog/
+.history
+.svn/
+
+# IntelliJ related
```

Comment:
Where is this being copied from? Can we clean up the unnecessary user specific ignores upstream before copying it?